src/Test.UnitTests.BinSkim.Driver/MultithreadedAnalyzeCommandTests.cs

```diff
@@ -121,15 +121,15 @@ public void MultithreadedAnalyzeCommand_ReturnCommonPathRootFromTargetSpecifiers
             foreach (var testCase in testCases)
             {
                 string commonPath = MultithreadedAnalyzeCommand.ReturnCommonPathRootFromTargetSpecifiersIfOneExists(testCase.TargetFileSpecifiers);
-                if (commonPath != testCase.ExpectedCommonPath)
+                if (!string.Equals(commonPath, testCase.ExpectedCommonPath, StringComparison.OrdinalIgnoreCase))
```

Comment:
>OrdinalIgnoreCase [](http://example.com/codeflow?start=93&length=17)

Tough problem, case-sensitivity in this scenario. This looks good as it would break cross-platform testing altogether if we every introduced case-sensitivity in this scenario. Thanks for the fix!